### ISSUE TITLE
test

### ISSUE THREAD

**Issue by Kristinita** [divinites/gissues#45]

**Comment by divinites** [comment]
Hi, Close and open issue is very simple with this plug-in. when you creat an issue, you get some thing like this:

```
    # Title         : test
    ## Number       : 48
    ## State        : open
    ## Locked       : False
    ## Assignee     : None
    *----------Content----------*
    test
    ## Add New Comment:

    *----------END----------*
*<Issue number 180995558 created at 2016-10-04T20:25:30Z>*
```

You can change "open" to "close" or revert "close" to "open", and choose `github issue: update issue`. The default keymap under Windows for `update issue` is `ctrl + s`.

If this is your repo, you can even change the assignee, or lock the issue to prevent further comments, and then update.

**Comment by divinites** [comment]
This is a bug... I tried to update a comment here, but it turns out to delete your original issue post... I think that it is a problem of line recognition. It is a serious bug, I will solve it right now.

**Comment by divinites** [comment]
solved by & ff7967972e52f8aee10cda0aa2f1f76615ae08b1 v.2.6.0
